history/characters/10000_orc.txt

```diff
@@ -10,7 +10,9 @@
 	trait=wrathful trait=greedy trait=arrogant trait=arbitrary
 	disallow_random_traits = yes
 	father = 10701 #Ugratil
-	542.1.13={ birth=yes trait=creature_orc effect = { make_important_lore_character_effect = yes } }
+	542.1.13={ birth=yes trait=creature_orc 
+		effect = { make_important_lore_character_effect = yes }
+	}
```

Comment:
Tab rules. If the opening and closing brackets aren't on the same line, the code between them goes on the next lines with a tabulation.
```suggestion
	542.1.13={
		birth=yes trait=creature_orc 
		effect = { make_important_lore_character_effect = yes }
	}
```
You may read it here, it's called K&R style.
https://en.wikipedia.org/wiki/Indentation_style#Brace_placement_in_compound_statements